docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst

```diff
@@ -18,54 +19,55 @@ stack, this is the method that you will almost definitely want to use.
 Enable EPEL
 -----------
 
+.. NOTE::
+
+   RHEL systems will need to enable the `EPEL Repositories`_ manually.
```

Comment:
@ayohrling I thought about that, but then we'd have to keep changing it as the link changes so I copped out to just passing them the full HOWTO.